index.html

```diff
@@ -322,7 +322,7 @@ <h2>Students just like you.</h2>
 		         <div class="container">
 		            <div class="col-md-10 col-md-offset-1 col-sm-7 col-sm-offset-1 v-align-transform text-center">
 		            
-                     <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி" , "സംഭാവനയ്ക്ക് നന്ദി"]'>
+                     <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.","ਯੋਗਦਾਨ ਪਾਉਣ ਲਈ ਤੁਹਾਡਾ ਧੰਨਵਾਦ", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி" , "സംഭാവനയ്ക്ക് നന്ദി"]'>
```

Comment:
Better to maintain the Order. There were 6 Languages already on the List. Don't add it in the Middle. Better to add it as seventh.